examples/test_xworld3d.cpp

```diff
@@ -36,7 +36,7 @@ int main(int argc, char** argv) {
     double reward_per_game = 0;
     double r = 0;
     while (true) {
-        // xwd->show_screen();
+        xwd->show_screen();
```

Comment:
Don't do this. People by default may not have any monitor to show.

Reply:
People who run these examples expects to see the screen. Printing only a few numbers is pointless. And now we use EGL, they should run this example to see if EGL works well.

Reply:
Even if with EGL, you still needs to ssh with -X. People do not usually do this.